src/network.c

```diff
@@ -329,6 +329,8 @@ static derive_t stats_values_sent = 0;
 static derive_t stats_values_not_sent = 0;
 static pthread_mutex_t stats_lock = PTHREAD_MUTEX_INITIALIZER;
 
+static int store_rates = 0;
```

Comment:
Can you please move this to line 288, next to the other variables pulled out of the configuration file ?